src/functionalTest/java/uk/gov/hmcts/reform/cwrdapi/CaseWorkerRefCreateTest.java

```diff
@@ -84,9 +81,97 @@ public void whenUserNotExistsInCwrAndSidamAndUp_Ac1_LD_disabled() {
                 .concat(FeatureConditionEvaluation.FORBIDDEN_EXCEPTION_LD));
     }
 
+    @Test
+    public void whenUserNotExistsInCwrAndUpAndExistsInSidam_Ac2() {
+        List<CaseWorkersProfileCreationRequest> profileCreateRequests = createNewActiveCaseWorkerProfile();
+        UserProfileResponse upResponse = getUserProfileFromUp(profileCreateRequests.get(0).getEmailId());
+        assertEquals(ImmutableList.of(CWD_USER, CASEWORKER_IAC_BULKSCAN), upResponse.getRoles());
+    }
+
+    @Test
+    public void whenUserExistsInCwrAndUpAndExistsInSidam_Ac3() {
+
```

Comment:
Just a suggestion, if we can put a comment like "//this test verifies that profiles are created in UP". 
It's quite difficult to grasp by "_Ac2,3,4" etc.

Reply:
put some comments